lib/badges/getUserBadgeTimestamp.js

```diff
@@ -0,0 +1,40 @@
+// Includes
+const http = require('../util/http').func
+
+// Args
+exports.required = ['userId','badgeId']
+
+// Define
+const getBadgeTimestamp = (userId, badgeId) => {
```

Comment:
Rename this method (The file and the function itself plus the typings) to getAwardedTimestamps.

I do not think there are any other assets that can be "awarded" and this avoids plurality surrounding "badge(s)" in the method name.

Reply:
Added